pyface/base_toolkit.py

```diff
@@ -187,7 +187,7 @@ def import_toolkit(toolkit_name, entry_point="pyface.toolkits"):
         If no toolkit is found, or if the toolkit cannot be loaded for some
         reason.
     """
-    entry_point_group = importlib_metadata.entry_points()[entry_point]
+    entry_point_group = importlib_metadata.entry_points().select(group=entry_point)
```

Comment:
Unfortunately, this code fails on newer Python versions, where `import importlib.metadata as importlib_metadata` is used: the return value from `entry_points()` is then a plain dict, which doesn't have a `select` method.

Reply:
I've opened #998 to track this. It might also be good to see if we can get the CI tests running on Python 3.9, so that this sort of failure is detected earlier.